cli/lib/kontena/cli/stacks/registry/show_command.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Registry
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::Stacks::Common
+    include Kontena::Cli::Stacks::Common::StackNameParam
+
+    banner "Shows information about a stack on the stacks registry"
+
+    option ['-v', '--versions'], :flag, "Only list available versions"
```

Comment:
`GET /stack/terom/redis` returns `Content-Type: text/x-yaml; charset=utf-8` and `Client#parse_response` returns the response body as a String.